.env.example

```
@@ -5,8 +5,8 @@ TRACE_ON=1
 START_AT_STEP=git-bake
 STOP_AT_STEP
 
-# Mount a local recipes from a different directory
-RECIPES_ROOT=./recipes
+# Mount a local cookbook/recipes from a different directory
+RECIPES_ROOT=./cookbook/lib/recipes
```

```suggestion
RECIPES_ROOT=./cookbook
```

All of the other definitions of `RECIPES_ROOT` in this PR point to `./cookbook`. This is just an example file